docs/exposing-binding-data/adding-annotation.md

```diff
@@ -1,6 +1,282 @@
 ---
-sidebar_position: 3
+sidebar_position: 4
 ---
 
 # Adding Binding Metadata as Annotations
 
```

Comment:
Change all the instances of 'inject' , 'injection' to: 'project', 'projection'

Reply:
done